3rdparty/openssl/CMakeLists.txt

```diff
@@ -70,20 +87,158 @@ endif ()
 # from git bash on Windows. For the consistency of the build flow, we keep the copies of files
 # generated by these scripts (see update.make) and directly use those copies on both Linux and Windows
 # builds.
+set(OPENSSL_CRYPTO_HEADERS bn_conf.h dso_conf.h)
+foreach (header_file ${OPENSSL_CRYPTO_HEADERS})
+  list(
```

Comment:
Let's create a macro for this.
For example
macro (append_copy_command LIST SOURCE DESTINATION)
  ...